release-notes/General/General_Feature_Release_10.5/General_Feature_Release_10.5.1.md

```diff
@@ -133,32 +135,32 @@ In a later iteration, the Swarming feature will also be able to assist in making
 
 > [!NOTE]
-> [Prerequisite checks](xref:EnableSwarming#running-a-prerequisites-check) are in place to prevent the enabling of the Swarming feature when non-supported objects are present. Where possible, you will also be prevented from configuring or creating these on a Swarming-enabled system.
+>
+> - Currently, Swarming is limited to [basic elements](xref:SwarmingElements). Support for other types of elements will be added in future versions.
+> - [Prerequisite checks](xref:EnableSwarming#running-a-prerequisites-check) are in place to prevent the enabling of the Swarming feature when non-supported objects are present. Where possible, you will also be prevented from configuring or creating these on a Swarming-enabled system.
 
 ##### Required user permissions
 
 To enable the Swarming feature, the [Admin Tools](xref:DataMiner_user_permissions#modules--system-configuration--tools--admin-tools) user permission is required.
 
-Once the feature has been enabled, users will need the [Swarming](xref:DataMiner_user_permissions#modules--swarming) user permission to trigger any swarming actions. Users that previously had the [Import DELT](xref:DataMiner_user_permissions#general--elements--import-delt) and [Export DELT](xref:DataMiner_user_permissions#general--elements--import-delt) user permissions before Swarming was enabled will automatically also get the *Swarming* user permission.
+Once the feature has been enabled, users will need the [Swarming](xref:DataMiner_user_permissions#modules--swarming) user permission to trigger any swarming actions. Users who previously had the [Import DELT](xref:DataMiner_user_permissions#general--elements--import-delt) and [Export DELT](xref:DataMiner_user_permissions#general--elements--import-delt) user permissions before Swarming was enabled will automatically also get the *Swarming* user permission when upgrading from a version without the *Swarming* permission to a version with the *Swarming* permission.
 
 To swarm an element, users will also need config rights on the element.
 
 ##### Swarming elements
 
 When Swarming has been enabled, you can swarm elements **in DataMiner Cube** via *System Center* > *Agents* > *Status*. On that page, the *Swarming* button will be displayed instead of the *Migration* button. Clicking the button will open a window where you can select the elements you want to swarm and the destination DMA.
 
-Swarming elements is also possible **via Automation**. For this, you will need an element key, i.e. the DMA ID/element ID pair of the element you want to swarm, and the target Agent ID. You can specify these as fixed values or as input variables. For example:
+Swarming elements is also possible **via Automation**. For this, you will to write a script with two input parameters: one for the element key, i.e. the DMA ID/element ID pair of the element you want to swarm, and another for the target Agent ID. For example:
```

Comment:
Maybe important to also note that for the script below, the names of the script input parameters must be "Element Key" and "Target Agent ID". Not sure if having these names in the script itself is clear enough here.